CHANGELOG_iOS.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.71.125](https://github.com/brave/brave-browser/releases/tag/v1.71.125)
+
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
+ - Added new Playlist UI to be default. ([#40911](https://github.com/brave/brave-browser/issues/40911))
+ - Added support for running iOS app in compatibility mode on VisionOS. ([#40579](https://github.com/brave/brave-browser/issues/40579))
```

Comment:
Should actually mention Brave rather than mentioning `iOS app`. For example:

```
 - Added support for running Brave in compatibility mode on VisionOS. ([#40579](https://github.com/brave/brave-browser/issues/40579))
```